core/templates/pages/exploration-editor-page/editor-tab/training-panel/training-data-editor-panel-modal.controller.spec.ts

```diff
@@ -181,7 +181,7 @@ describe('TrainingDataEditorPanelServiceModalController', function() {
         expect($scope.answerGroupHasNonEmptyRules).toBe(true);
         expect($scope.inputTemplate).toBe(
           '<oppia-interactive-text-input ' +
-          'label-for-focus-target="testInteractionInput" last-answer="null">' +
+          'last-answer="null" label-for-focus-target="testInteractionInput">' +
```

Comment:
Why this change?

Reply:
I've added another attribute while generating the interaction HTML which is probably why this change is required. This is just reordering of the terms so it doesn't seem like a major issue.

Reply:
Sure, I was just curious.